### PR TITLE
Sundry fixes

### PR DESCRIPTION
**Pull request by mruwnik**
e.g. 
```
requests.get(
    'https://aistampy.info/questions/allQuestions',
    auth=HTTPBasicAuth('stampy', 'stampy'),
    params={'dataType': 'singleFileJson', 'questions': 'inProgress'}
).json()
```

### CODE REVIEW

File: app/components/search.tsx [hunk 1]
```diff
@@ -31,6 +33,17 @@ export default function Search({limitFromUrl, className}: Props) {
     }
   }
 
+  useEffect(() => {
+    const query = searchParams.get('q')
+    if (loadedQuestions && query) {
+      searchFn(query)
+      setShowResults(true)
+    }
+    // Properly adding all dependancies would require transforming `searchFn` into a callback
```
Comment (Aprillion) [inline app/components/search.tsx:42]: from phone, but something like:
```ts
const searchFnRef = useRef(searchFn)
searchFnRef.current = searchFn
useEffect(
...searchFnRef.current(),
...
```

File: wrangler.toml.template [hunk 2]
```diff
@@ -25,4 +25,4 @@ MATOMO_DOMAIN = "{MATOMO_DOMAIN}"
 DISCORD_LOGGING_CHANNEL_ID = "{DISCORD_LOGGING_CHANNEL_ID}"
 DISCORD_LOGGING_TOKEN = "{DISCORD_LOGGING_TOKEN}"
 EDITOR_USERNAME = "{EDITOR_USERNAME}"
-EDITOR_PASSWORD = "{EDITOR_PASSWORD}"
+EDITOR_PASSWORD = "{EDITOR_PASSWORD}"
```
Comment (Aprillion) [inline wrangler.toml.template:28]: ?

**Review by Aprillion — APPROVED** [review]

**Review by LeMurphant — APPROVED** [review]
I don't know about the code, but the format to request the pages looks good.

File: wrangler.toml.template (hunk 2 above)
Comment (mruwnik) [reply wrangler.toml.template:28]: trailing whitespace

File: app/components/search.tsx (hunk 1 above)
Comment (mruwnik) [reply app/components/search.tsx:42]: dunno if that's better... :D

File: wrangler.toml.template (hunk 2 above)
Comment (Aprillion) [reply wrangler.toml.template:28]: yes, why removing it?

File: app/components/search.tsx (hunk 1 above)
Comment (Aprillion) [reply app/components/search.tsx:42]: disabling the eslint rule is unsafe for future changes which can more easily [introduce bugs](https://react.dev/learn/removing-effect-dependencies#to-change-the-dependencies-change-the-code).. mutable refs might not look good, but you explicitly say "this is not a dependency which triggers a synchronization, I just want to use whatever value is has currently" without implying anything about any other variable that may or may not be a dependency

**Pull request merged**